int32 sender_robot_id
int32 local_vertex_index
float64[4] pose_pudq
float64[3] pose_vec3
